x-pack/qa/transport-client-tests/src/test/java/org/elasticsearch/xpack/ESXPackSmokeClientTestCase.java

```diff
@@ -67,6 +70,9 @@ private static Client startClient(Path tempDir, TransportAddress... transportAdd
                 .put("client.transport.ignore_cluster_name", true)
                 .put("xpack.security.enabled", false)
                 .put(Environment.PATH_HOME_SETTING.getKey(), tempDir);
+        if (Boolean.parseBoolean(System.getProperty(FIPS_SYSPROP)) && JavaVersion.current().getVersion().get(0) == 8) {
```

Comment:
Same here?

```suggestion
        if (inFipsSunJsseJvm()) {
```

Reply:
see https://github.com/elastic/elasticsearch/pull/52211#issuecomment-585092157

Reply:
This needs a comment in code (and preferably a tracking issue) or someone may inadvertently try to make the same obvious change.

Reply:
yes, that makes total sense - I'll make do